staff/jakub-gaba/course-1/contact.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<meta charset="utf-8">
+<title>Contact</title>
+    <link rel="stylesheet" href="main.css">  
+</head>
+<body>  
+  <div class="bigboss">
+  <div class="container_logo">      
+      <div class="text_logo">
+        <img class="logo" src="LOGO.png"/>
+        <h1>NorCal Time Trial Series
+          <nav class="nav">
+            <a class="nav__item" href="index.html">Home </a><element class="line">|</element>
+            <a class="nav__item" href="about.html">About NYSL</a> <element class="line">|</element>
+            <a>Contact</a>
+          </nav>
+        </h1>
+      </div>
+    </div>
+    <br>
+    <div class="container">      
+        <img class="images" src="Image 3.png"/>
+        <div class="contact_text">
+          <br>
+        Please email us at: <a class="nav__item" href="http://">info@nctts.org</a><br>We will
```

Comment:
To make email works use a mailto tag, documentation: https://www.w3schools.com/tags/tag_address.asp